nmi_mysql/nmi_mysql.py

```diff
@@ -134,6 +149,8 @@ def _execute(self, _query, _params, _multi=False):
                         'affected_rows': result.rowcount
                     }
 
+            con.close()
```

Comment:
Okay then, I guess there's really no need for the client to cache the connection since it is already handled by the pool. So I guess we should close here. (Though I'm wondering what would happen if the client close/release the connection again after we have already closed it?) if this works without exception/error then I think we can leave this as is.